CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 - Move Hooks and Silenced out of Event and into Check.
 - Handle round-robin scheduling in wizardbus.
 - Added informational logging for failed entity keepalives.
+- Replaced fileb0x with vfsgen for bundling static assets into binary.
```

Comment:
Do you mind adding an entry to indicate that the dashboard is now automatically built when building the backend?

Reply:
Good call. I should probably update the README as well.